src/py/flwr/server/superlink/state/sqlite_state.py

```diff
@@ -63,7 +64,8 @@
 CREATE TABLE IF NOT EXISTS run(
     run_id          INTEGER UNIQUE,
     fab_id          TEXT,
-    fab_version     TEXT
+    fab_version     TEXT,
+    overrides       TEXT
```

Comment:
Let's use the same name everywhere:
```suggestion
    override_config       TEXT
```